tasks/travis-pr-title-lint.js

```diff
@@ -154,7 +154,7 @@ module.exports = function(grunt) {
 
       Promise.all([
         getPreset('angular'),
-        getConfiguration('.conventional-changelog-lintrc'),
```

Comment:
Oh man, I can't believe they don't raise an error for this. How about adding some annoying code to check the file since they don't raise an error? Something like:
````javascript
var config = path.join(__dirname, '..', '.conventional-changelog-lintrc');
var configStat = file.statSync(config);
if (!configStat || !configStat.isFile()) {
  throw new Error(...);
}
````